docs/quickstart/share-report-ui.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Share Data Observability Report"
+title: "Share data observability report"
```

Comment:
I know I changed this, but it looks strange with the above headline.
Let's change to:
Share observability report